### DISCUSSION TITLE
SSL / HTTPS ability

### DISCUSSION THREAD

**Discussion by Emilien1** [prasathmani/tinyfilemanager#843] [Q&A]
Hello,

This is a very nice tool.
I have been trying to add certificate to the service for configuration of the SSL layer.
Is there any options for it ?

Many Thanks, 

Emilien

**Comment by cr0n-dist0rti0n** [comment]
You have to HTTPS the whole site. You can't just do an individual page. You can checkout https://certbot.eff.org for more information on how to do that.
